install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -158,13 +158,9 @@ openshift_kuryr_precreate_subports: 5
 
 kuryr_openstack_public_net_id: *<public_ID>*
 
-# To enable namespace isolation, uncomment
-#openshift_kuryr_subnet_driver: namespace
-#openshift_kuryr_sg_driver: namespace
-
-# Select kuryr image (always latest available)
-openshift_openstack_kuryr_controller_image: registry.access.redhat.com/rhosp14/openstack-kuryr-controller:latest
-openshift_openstack_kuryr_cni_image: registry.access.redhat.com/rhosp14/openstack-kuryr-cni:latest
+# To disable namespace isolation, comment the next 2 lines
```

Comment:
s/comment/comment out

> ! comment out v
Use to describe the act of adding characters to the beginning of a line of code to indicate that the line is not
to be compiled.